Bottom: 2b8c6771202830d532236700dde790e651edc2a9
Top:    129c5b6b3dfa3e1d83e8cd7a7320ed1a628cc40f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:31:26 +0000

Turn off debug hacks in object generation


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 52bb954..f8d7304 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1531,9 +1531,9 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   destroy_appearance (copper_appearance);
 
   /* DEBUG */
-  poly_M_Copy0 (&PCB->Data->outline, info.poly);
-  PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  poly_M_Copy0 (&PCB->Data->outline, info.poly);
+//  PCB->Data->outline_valid = true;
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
